akid/akids.go

```diff
@@ -8,31 +8,32 @@ import (
 )
 
 const (
-	APISpecTag            = "api"
-	APIKeyTag             = "apk"
-	APIMethodTag          = "mth"
-	ClientTag             = "cli"
-	ConnectionTag         = "cxn"
-	DataCategoryTag       = "dct"
-	GraphTag              = "gph"
-	IdentityTag           = "idt"
-	InvalidTag            = "xxx"
-	LearnSessionTag       = "lrn"
-	MessageTag            = "msg"
-	MonitorTag            = "mtr"
-	OrganizationTag       = "org"
-	OutboundRequestTag    = "obr"
-	ProjectTag            = "prj"
-	RequestTag            = "req"
-	RuleTag               = "rul"
-	ScheduleTag           = "sch"
-	ServiceClusterTag     = "scl"
-	ServiceTag            = "svc"
-	ShardAliasTag         = "sal"
-	ShardTag              = "shd"
-	UsageTrackingEventTag = "ute"
-	UserTag               = "usr"
-	WitnessTag            = "wit"
+	APISpecTag             = "api"
+	APIKeyTag              = "apk"
+	APIMethodTag           = "mth"
+	ClientTag              = "cli"
+	ConnectionTag          = "cxn"
+	DataCategoryTag        = "dct"
+	GraphTag               = "gph"
+	IdentityTag            = "idt"
+	InvalidTag             = "xxx"
+	LearnSessionTag        = "lrn"
+	MessageTag             = "msg"
+	MonitorTag             = "mtr"
+	OrganizationTag        = "org"
+	OutboundRequestTag     = "obr"
+	PathParameterPrefixTag = "pre"
```

Comment:
(Non-blocking.) How about something more distinctive? There might be other kinds of prefixes. Granted, it's hard to compress "path parameter prefix" into just three letters.
```suggestion
	PathParameterPrefixTag = "ppx"
```